nautobot/core/templates/403_csrf_failure.html

```diff
@@ -0,0 +1,9 @@
+{% extends '40x.html' %}
+
+{% block title %}Invalid Form Submission{% endblock %}
+
+{% block icon %}<i class="mdi mdi-alert"></i>{% endblock %}
+
+{% block message %}
+    CSRF failure has occurred.
```

Comment:
Is there not a message that comes from the exception that is raised that is possibly more descriptive here?

Also could we not overload  the `more` context variable used in the [default template](https://github.com/django/django/blob/main/django/views/csrf.py)? vs. introducing another template to manage?

Reply:
See above responses and my first comment. 

Happy to add reason to the output.

Reply:
I think that would be a nice middle ground. to say CSRF failed and at least attempt to explain why unless we provide a generic reason like "Refresh the page and try again." or something to that effect.

Reply:
Added in reason and generic boilerplate from the 500.html template